frontend/src/components/MissionOverview/MissionCard.tsx

```diff
@@ -0,0 +1,5 @@
+import { Card, Typography } from '@equinor/eds-core-react'
+
+export function MissionCard() {
+    return <Card>hello</Card>
```

Comment:
Remove hello as this makes it so it says three hello three times below all three of the cards.

Reply:
This is expected behaviour